content/news/2018-07-30-navpi-memory-management-v1-0-9-release-notes.md

```diff
@@ -0,0 +1,83 @@
+
+### The problem; memory is maxed
+
+When the NavCoin daemon starts, it loads the entire chainstate database in memory. This causes a large spike in memory usage until that operation is complete. Existing users have reported that after updating to the latest version of NavCoin, the memory spike can cause the Chromium WebUI to be nearly unusable in addition to the navcoind daemon simply crashing. 
+
+### The short term solution; add swap space
+
+Creating dedicated swap space was the key to resolving the memory issue, and three approaches; zram swap, usb swap, and sd swap.
+
+### What is the function of swap space?
+
+The swap space is an area on the Raspberry Pi’s SD card (or additional USB drive) intended as a place the operating system can store data temporarily.  
+
+### How does NavPi use swap space?
+
+NavPi uses the swap space to increase the amount of information it can process “in memory” while syncing the blockchain and performing other essential operations.
+
+### ZRAM swap
+
+The initial approach to the problem was to enable ZRAM on the NavPi as a temporary fix. For many users it seemed to help, but during the core team’s testing cycles there were still resource issues on the initial load of the chain state in memory. 
+
+* [How to enable ZRAM swap on NavPi](https://info.navcoin.org/knowledge-base/enable-zram-navpi/) (NavCoin Knowledge Base)
+
+### SD swap
+
+As another temporary solution, we created an article on the Knowledge Base demonstrating how to enable the swap space on the SD card of the NavPi. The steps in the article were based on an article written by Justin Ellingwood of Digital Ocean and tested successfully on NavPi v1.0.8. While this approach did result in added stability for both the NavCoin daemon and the Chromium WebUI, it is not our recommended approach long-term.
+
+* [How to enable SD swap space on NavPi](https://info.navcoin.org/knowledge-base/navpi-sd-swap/) (NavCoin Knowledge Base)
+
+### USB swap
+
+For best results, it’s recommended to utilize a USB drive as your swap space, due to the fact that repeated reads/writes to the SD card can potentially wear it out. There are multiple articles, posts and resources from the linux and Raspbian community demonstrating how to setup a swap space on a USB. We tested many approaches and created our own version intended for the NavPi, which is now documented Knowledge Base.
+
+* [How to enable USB swap space on NavPi](https://info.navcoin.org/knowledge-base/navpi-usb-swap/) (NavCoin Knowledge Base)
+
+### Long-term solutions
+
+There are plans to enhance the NavPi to remedy the memory issue long term, including: 
+
+* optimise NavCoin core (levelDB)
```

Comment:
I think this list should be capitalised:
- Optimise NavCoin Core (**specifically it's levelDB databases**)
- Optimise **the** NavPi OS 
- Optimise NavPi application (headless)   <-- I'm not actually sure what you mean by this, turn the OS headless?
- Upgrade to **the new** Rasbpian OS (Stretch) **in order to support the** new RaspberryPi 3b+ model

Reply:
fixed